src/_posts/2017/2017-04-01-statie-generate-and-refresh.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: post
+title: 'Statie: generate & refresh'
+description: '''
+Make development with <a href="https://github.com/Symplify/Statie">Statie</a> a bit more convenient with a little use of <a href="http://gulpjs.com/">gulp</a> and <a href="https://browsersync.io/">Browsersync</a>. Take a look at how to implement <a href="http://lesscss.org/">Less</a> preprocessor in the building workflow.
+'''
+---
+
+With a rising popularity of static site generators, we can find that there are [many of them][1] already in use. This blog runs on [Statie][2], another such a tool written by [Tomas Votruba](https://www.tomasvotruba.cz/), which differs from the others with [Nette](https://nette.org/) and its templating engine [Latte](https://latte.nette.org/) used in the background.
+
+If you are not aware what is this tool all about let's take a look at a post series about it on [Tomas' blog][3] firstly. It will lead you from the very beginning on how to run your own site upon Statie, so I won't deal with this here.
+
+In the end of the mentioned blog post, we can find a [simple gulp task](https://www.tomasvotruba.cz/blog/2017/02/20/statie-how-to-run-it-locally/#minitip-use-gulp-work-for-you) that runs generate command on every change in the source files. This is surely handy, but you still need to refresh your browser to be able to see your changes. With **Browsersync** we can make this step unnecessary.
+
+## Running an executable command
+With `vendor/bin/statie generate` command we can generate our site content. In `gulpfile.js` we may have something like this:
+
+``` javascript
+var gulp = require('gulp');
+var exec = require('child_process').exec
+
+gulp.task('default', function () {
+	exec('vendor/bin/statie generate');
+});
+```
+
+The problem is it doesn't give you a message when the job is done, so you cannot tell when to refresh a browser to see new content. To get over this problem [gulp-run](https://github.com/MrBoolean/gulp-run) package may be used.
```

Comment:
"The problem is it doesn't give you a message when the job is done, so you cannot tell when to refresh a browser to see new content"

This could be in **bold**. It seems important